staging/src/k8s.io/kubectl/pkg/cmd/wait/wait_test.go

```diff
@@ -1046,6 +1046,18 @@ func TestWaitForDifferentJSONPathExpression(t *testing.T) {
 
 			expectedErr: "timed out waiting for the condition on theresource/foo-b6699dcfb-rnv7t",
 		},
+		{
+			name: "JSONPath condition not given",
+			fakeClient: func() *dynamicfakeclient.FakeDynamicClient {
+				fakeClient := dynamicfakeclient.NewSimpleDynamicClientWithCustomListKinds(scheme, listMapping)
+				fakeClient.PrependReactor("list", "theresource", listReactionfunc)
+				return fakeClient
+			},
+			jsonPathExp:  "{.status.containerStatuses[0].ready}",
+			jsonPathCond: "",
+
+			expectedErr: None,
+		},
```

Comment:
nit: Consider adding a test for "empty" value that is currently not implemented (i.e. `{.status.containerStatuses[0].ready}=`).

Reply:
Yes, there is a clearly missing test that validates the correct parsing. It should handle the expected error when passing "invalid" json expression